README.md

```diff
@@ -19,14 +19,20 @@
 
 CRI-O follows the Kubernetes release cycles with respect to its minor versions
 (`1.x.y`). Patch releases (`1.x.z`) for CRI-O are not in sync with those from
-Kubernetes, because those are scheduled for each month, whereas CRI-O provides
+Kubernetes, because they are scheduled for each month, whereas CRI-O provides
```

Comment:
hah same note as in cri-tools https://github.com/kubernetes-sigs/cri-tools/pull/1246/files#r1300154003

Reply:
I'm fine as a follow up too

Reply:
Yep, fixed as suggested.